iOSClient/Files/NCFiles.swift

```diff
@@ -175,7 +175,7 @@ class NCFiles: NCCollectionViewCommon {
         isReloadDataSourceNetworkInProgress = true
         collectionView?.reloadData()
 
-        networkReadFolder(isForced: isForced) { tableDirectory, metadatas, metadatasUpdate, metadatasDelete, error in
+        networkReadFolder(isForced: isForced) { tableDirectory, metadatas, differentCount, metadatasChanged, error in
```

Comment:
why do we need `differentCount`? Can't we just use `metadatasChanged.count`?

Reply:
I see it's a bool. Can we then rename `differentCount` to `metadatasChangedCount`? In that case we would not need the `metadatasChanged` bool since you can just do `metadatasChangedCount == 0`?